src/llamacrawl/services/readers/elasticsearch.py

```
@@ -521,3 +523,40 @@ def get_api_client(self) -> Elasticsearch:
             Elasticsearch client
         """
         return self.es_client
+
+    @retry_with_backoff(max_attempts=3)
+    def _open_point_in_time(self, index: str, keep_alive: str) -> Any:
```

Ruff (0.14.0)

Dynamically typed expressions (typing.Any) are disallowed in `_open_point_in_time`

(ANN401)